src/Bundle/DependencyInjection/DunglasDoctrineJsonOdmExtension.php

```diff
@@ -50,5 +50,8 @@ public function load(array $configs, ContainerBuilder $container): void
         if (!class_exists(BackedEnumNormalizer::class) || !class_exists(\BackedEnum::class)) {
             $container->removeDefinition('dunglas_doctrine_json_odm.normalizer.backed_enum');
         }
+
+        $config = $this->processConfiguration(new Configuration(), $configs);
+        $container->setParameter('dunglas_doctrine_json_odm.type_map', $config['types']);
```

Comment:
Couldn't we create a `TypeMapper` service instead? By using an optional service (null if this option isn't used), we'll probably also improve the performance by preventing some function calls on the hot path.